app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -28,11 +29,11 @@ class AuthRepositoryTest : ShouldSpec(
 			unmockkAll()
 		}
 
-		listOf(
-			null to AuthEvent.NotLogged,
-			mockk<FirebaseUser>() to AuthEvent.Logged,
-		).forEach { (user, authEvent) ->
-			should("return $authEvent WHEN observeEvents called AND AuthStateListener returned $user user") {
+		should("return authEvent depending on user returned from AuthStateListener WHEN observeEvents called") {
```

Comment:
i think that we also agreed on using forAll, to have one tests for values instead of few tests